dictionaries/noty.definition.json

```diff
@@ -56,6 +56,12 @@
     "user_suspended_title": {
         "en": "Identity suspended"
     },
+    "user_error": {
```

Comment:
There is a problem with your identifier (%ORGID%). Please get in touch with our support team at <a href="mailto:support@rciam.example.org">support@rciam.example.org</a> and include the error message shown above. We apologise for any inconvenience caused and will help you resolve this issue as quickly as possible.

Reply:
Done